### DISCUSSION TITLE
SabNZB, Organizr, not working

### DISCUSSION THREAD

**Discussion by 1d10t5** [htpcBeginner/docker-traefik#163] [Q&A]
I am probably missing something, but I've managed to get Traefik, OAuth, Lidarr, Sonarr, Radarr, Heimdall, Lazy Librarian, CF-comp, and CF-DDNS all working.

I cannot get SabNZB nor Organizer to be reachable. Traefik shows no errors, and I really don't know where to look to figure out what I'm doing wrong.

Even a little nudge in the right direction would be super awesome and helpful.  I know everyone here has their own lives and are busy, and I'm patient.  I've only been adding things one at a time to make sure things work before continuing.  Worked well until Organizer and SabNZB.

Thank you in advance for any and all advice.

**Comment by raulvasquez** [comment]
If you expose the ports, and you use local ip and reach it? You need to disable the built-in https support in sabnzbd's settings also.

**Reply by 1d10t5** [reply]
Thank you for the reply.

I will try that, but I have been doing all this remote, through a domain name with subdomains for each service. I have even exposed a raspberry pi as well as the ubuntu server, for management, through traefik on subdomains. 

I'm not sure where or what setting for sabnzbd you are referring to. Is this in the docker-compose yml? If so, please elaborate.

I did decide to try nzbget last night and succeeded without any issues.

Also, any idea about organizr? That's really more important to me.

Thank you again.
